### ISSUE TITLE
[Feature Request] Changing backgrounds

### ISSUE THREAD

**Issue by breiti-oi** [fennifith/Alarmio#50]

**Comment by fennifith** [comment]
The app already has fairly robust dynamic theming, so it shouldn't be technically difficult to integrate this. The only thing I'd be worried about is overcomplicating the settings UI.

If I eventually add the ability for users to define their own custom themes, then I could allow them to specify a different background image for each one. This, combined with the "schedule" option to switch between the day/night themes at certain times, could effectively allow the functionality that you describe. It will be a while before I start to work on that, though, so this may be left open for some time.

**Comment by breiti-oi** [comment]
> The only thing I'd be worried about is overcomplicating the settings UI. 

You're right about this. In mentioned Omniclock you can only choose wether to show an background when the alarm rings, or not. The app itself "decide" which one is best for this time of the day. So you got one for nighttime, one for sunrise and one for morning.
